packages/manager/apps/dedicated/client/app/account/otrs/otrs.app.js

```diff
@@ -1,4 +1,4 @@
-import ovhManagerCore from '@ovh-ux/manager-core';
+import angular from 'angular';
```

Comment:
why did you remove  import ovhManagerCore from '@ovh-ux/manager-core';

Reply:
It has been removed by mistake, i have not touched otrs and contacts